storm-client/src/storm.thrift

```diff
@@ -206,10 +206,8 @@ struct NimbusSummary {
 
 struct ClusterSummary {
   1: required list<SupervisorSummary> supervisors;
-  //@deprecated, please use nimbuses.uptime_secs instead.
-  2: optional i32 nimbus_uptime_secs = 0;
-  3: required list<TopologySummary> topologies;
-  4: required list<NimbusSummary> nimbuses;
+  2: required list<TopologySummary> topologies;
+  3: required list<NimbusSummary> nimbuses;
```

Comment:
Can we please follow the guidelines for maintaining wire compatibility?  this way old clients still have a chance of working with the newer servers.

```
struct ClusterSummary {
   1: required list<SupervisorSummary> supervisors;
   // removed (DO NOT REUSE) 2:
   3: required list<TopologySummary> topologies;
   4: required list<NimbusSummary> nimbuses;
}
```

Reply:
Sure. Do you have a link to the guidelines, I didn't see anything in DEVELOPER.md?

Reply:
Thanks. Is it worth it to link to in DEVELOPER,md, or somewhere else do you think?